internal/configuration/validator/session.go

```diff
@@ -218,7 +218,9 @@ func validateRedis(config *schema.SessionConfiguration, validator *schema.Struct
 
 	validateRedisCommon(config, validator)
 
-	if !path.IsAbs(config.Redis.Host) && (config.Redis.Port < 1 || config.Redis.Port > 65535) {
+	if config.Redis.Port == 0 {
+		config.Redis.Port = 26379
```

Comment:
This should be the correct redis default port which is 6379, not 26379. Also this should utilize the schema.DefaultRedisConfiguration as the value should be set and used from there..